nuxeo-features/nuxeo-platform-audit/nuxeo-platform-audit-api/src/main/java/org/nuxeo/ecm/platform/audit/api/AuditReader.java

```diff
@@ -203,17 +203,13 @@
      *
      * @since 9.3
      */
-    default long getLatestLogId(String repositoryId, String... eventId) {
-        throw new UnsupportedOperationException("getLatestLogId is not yet implemented in this audit backend");
-    }
+    long getLatestLogId(String repositoryId, String... eventId);
```

Comment:
Could you rename `eventId` to `eventIds` here and in the implementations of the method, to clarify that it's an array?